BrowserServicesKit/Sources/Subscription/SubscriptionURL.swift

```diff
@@ -25,12 +25,14 @@ public enum SubscriptionURL {
     case baseURL
     case purchase
     case faq
+    case activationFlow
```

Comment:
New activation flow URL + two variants that skip to particular steps in the flow as required in some of the use cases